src/Service/Service.php

```diff
@@ -29,14 +29,14 @@
      *
      * @var HttpClient
      */
-    private $httpClient;
+    protected $httpClient;
```

Comment:
it's be ok. We can leave visibility for properties httpClient and requestFactory. And change only request method. I add alternative pull request #9